db/r2ghidra-sleigh

```diff
@@ -5,7 +5,7 @@ R2PM_DESC "Ghidra Sleigh Processors, required for r2ghidra"
 
 R2PM_DOC=""
 USE_BIN=1
-R2G_VERSION=5.5.2
```

Comment:
Why this change appears in app your prs?

Reply:
i mean, this change was already in previous pullreqs you submitted